tests/Builder/Accumulator/ExpMovingAvgAccumulatorTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MongoDB\Tests\Builder\Accumulator;
+
+use MongoDB\Builder\Accumulator;
+use MongoDB\Builder\Expression;
+use MongoDB\Builder\Pipeline;
+use MongoDB\Builder\Stage;
+use MongoDB\Tests\Builder\PipelineTestCase;
+
+use function MongoDB\object;
+
+/**
+ * Test $expMovingAvg accumulator
+ */
+class ExpMovingAvgAccumulatorTest extends PipelineTestCase
+{
+    public function testExponentialMovingAverageUsingAlpha(): void
+    {
+        $pipeline = new Pipeline(
+            Stage::setWindowFields(
+                partitionBy: Expression::stringFieldPath('stock'),
+                sortBy: object(
+                    date: 1,
+                ),
+                output: object(
+                    expMovingAvgForStock: Accumulator::expMovingAvg(
+                        input: Expression::numberFieldPath('price'),
+                        alpha: 0.75,
+                    ),
+                ),
+            ),
+        );
+
+        $this->assertSamePipeline(Pipelines::ExpMovingAvgExponentialMovingAverageUsingAlpha, $pipeline);
+    }
+
+    public function testExponentialMovingAverageUsingN(): void
+    {
+        $pipeline = new Pipeline(
+            Stage::setWindowFields(
+                partitionBy: Expression::stringFieldPath('stock'),
+                sortBy: object(
+                    date: 1,
+                ),
+                output: object(
+                    expMovingAvgForStock: Accumulator::expMovingAvg(
+                        input: Expression::numberFieldPath('price'),
+                        N: 2,
```

Comment:
I noted that `$expMovingAvg` indeed uses an uppercase single-letter parameter name. I'm sure that won't be confusing for people. No need to do something about this now, but I can see ourselves adding some kind of fallback for folks that aren't used to this and use `n` out of habit.

Reply:
PHP is case-insensible for method names but not for property or argument names. Static analysis and PHP error will help:
> Fatal error: Uncaught Error: Unknown named parameter $n

Reply:
Yep, that will indeed help :D